.readme-partials.yml

```diff
@@ -80,6 +80,10 @@ body: |-
 
   You may also want to see the [@google-cloud/error-reporting][@google-cloud/error-reporting] module which provides direct access to the Error Reporting API.
 
+  ### Special Payload Fields in LogEntry
+
+  There are some fields that are considered special by Google cloud logging and will be extracted into the LogEntry structure. For example, `severity`, `message` and `labels` can be extracted to LogEntry if included in the bunyan log payload. These [special JSON fields](https://cloud.google.com/logging/docs/structured-logging#special-payload-fields) will be used to set the corresponding fields in the `LogEntry`. Please be aware of these special fields to avoid unexpected logging behavior. 
```

Comment:
I believe this only applies to logs written to stdout, when parsed through the Logging Agent. You should make sure that is captured in the message here

Reply:
Sorry Daniel I missed this comment earlier. 

Initially I thought the same, then I looked into code and it looks like we are honoring these special fields even without `redirectToStdout: true`. Here is the where the logEntry is formatted: https://github.com/googleapis/nodejs-logging-bunyan/blob/main/src/index.ts#L253.

An example code snippet I was using Bunyan logger without `redirectToStdout`:

`const loggingBunyan = new LoggingBunyan();

const logger = bunyan.createLogger({
  name: 'my-service',
  streams: [
    {stream: process.stdout, level: 'info'},
    loggingBunyan.stream('info'),
  ],
});

logger.info({ message: 'boom11',  labels: {first: "label-name", count: "2"} }, 'this is the actual message');
`

You can see the special fields (`traceSampled` and `labels` here) are being processed and actually promoted to the log entry level: 
<img width="880" alt="image" src="https://github.com/googleapis/nodejs-logging-bunyan/assets/148148319/fbcd65f2-0d0e-4169-9a1c-4afdf1ba5eff">

Reply:
Ok interesting, I was thinking from the backend perspective, but I guess nodejs must be doing its own parsing as well. LGTM then